### PR TITLE
Update generated protos - add openconfig-system.

### PR DESCRIPTION
**Pull request by robshakir**
```
 * (M) proto/generate.sh
   - Update generate.sh to avoid cloning YangModels/yang which has
     become very large. Directly use curl to get revisions of modules
     that are expected by OpenConfig.
 * (M) proto/.../gen.go
   - Fix package names - consider explcitly specifying go_package in
     the generated protobuf.
 * (M) proto/.../*.proto
   - Regenerate protobufs.
```

### CODE REVIEW

File: proto/openconfig/openconfig_interfaces/gen.go [hunk 1]
```diff
@@ -15,7 +15,7 @@
 
 // Package openconfig_interfaces contains code generated from the OpenConfig
 // Interfaces protobuf for Hercules.
-package openconfig_interfaces
+package openconfig_openconfig_interfaces
```
Comment (wmohsin) [inline proto/openconfig/openconfig_interfaces/gen.go]: Trying to understand what leads to this doubling of "openconfig_" for most/all packages, and if anything can be done about this from a readability POV (it looks a bit odd/artificial) ? Apologies if this was discussed previously and I just cannot recall it. Can the tool be taught to suppress the first openconfig_, or does it lead to unwanted implications?

**Comment by robshakir** [comment]
This comes from the fact that we have the protocol name
openconfig.openconfig_interfaces - we can remove the stuttering by adding
an explicit go package name statement in the generated proto.

I filed ygot issue 208 (https://github.com/openconfig/ygot/issues/208) to
track this.

r.

File: proto/openconfig/openconfig_interfaces/gen.go (hunk 1 above)
Comment (aghaffarkhah) [reply proto/openconfig/openconfig_interfaces/gen.go]: This seems to be affecting the go files only, I see the proto files are following the previous naming approach.

Comment (robshakir) [reply proto/openconfig/openconfig_interfaces/gen.go]: Yes - this was just an error in the package naming that came to light when I regenerated the files. It's just for Go -- going forward setting the `go_package` explicitly will remove this stuttering.

**Pull request merged**